indico/modules/events/editing/service.py

```diff
@@ -78,10 +81,39 @@ def service_handle_enabled(event):
         'title': event.title,
         'url': event.external_url,
         'token': editing_settings.get(event, 'service_token'),
+        'config_endpoints': {
+            'tags': {
+                'create': url_for('.api_create_tag', event, _external=True),
+                'list': url_for('.api_tags', event, _external=True),
+                'tag_actions': {
+                    tag.code: {
+                        'update': url_for('.api_edit_tag', event, tag_id=tag.id, _external=True),
+                        'delete': url_for('.api_edit_tag', event, tag_id=tag.id, _external=True)
+                    } for tag in event.editing_tags
+                }
+            },
+            'editable_types': url_for('.api_enabled_editable_types', event, _external=True),
+            'file_types': {
+                t.name: {
+                    'create': url_for('.api_add_file_type', event, type=t.name, _external=True),
+                    'list': url_for('.api_file_types', event, type=t.name, _external=True),
+                    'file_type_actions': {
+                        ft.name: {
```

Comment:
Not sure if using the name as a key is a good idea here, since this cannot be used as a stable identifier (it's the user-facing name which can be changed at any time).

Actually, I'm wondering whether sending those `<tag|file_type>_actions` here is a good idea to begin with. The service would not only have to store this data (should we add an endpoint the service can use to retrieve this kind of config data again?) but also lack any information about tags/filetype created after enabling the service.

Maybe it'd be better to just send the URLs that never change (ie create and list), and include the *update* and *delete* URLs when retrieving the actual tags/filetypes via the *list* url? Looks like you already added the `url` to the schemas anyway

Reply:
I think that makes sense. I believe that was actually my original approach.